src/OrchardCore.Modules/OrchardCore.ContentTypes/Views/Admin/ListParts.cshtml

```diff
@@ -18,14 +18,16 @@
     </div>
 </div>
 
-<ul id="list-items" class="list-group">
-    @foreach (var type in Model.Parts)
-    {
-        <li class="list-group-item" data-filter-value="@type.DisplayName" data-type-name="@type.Name">
-            @Html.DisplayFor(m => type)
-        </li>
-    }
-</ul>
+<form method="post">
```

Comment:
Why form here? Look how we handle the delete button elsewhere

Reply:
No need to add a form for each entry

Reply:
I don't think it's a good idea. Each button should have its own form or use what we're use everywhere else.

Reply:
Me too, that's why I placed here to become a single form for all the buttons

Reply:
Yes it'll work. But it's invalid usage. If someone want to injects a form in one row, it may cause HTML issue since you'll end up with nested forms. So I don't think we should add a form here as a wrapper for all of the rendered records

Reply:
I didn't get your point. Please let me know what you are proposing here

Reply:
Remove the added form element

Reply:
Follow what we do elsewhere. For example here: https://github.com/OrchardCMS/OrchardCore/blob/main/src/OrchardCore.Modules/OrchardCore.Contents/Views/ContentsButtonActions_SummaryAdmin.cshtml#L40

Reply:
If I'm not wrong, I tried at the beginning but something weird happened. I will try to do it again to finalize this PR